docs/pages/web/chartgraph.js

```diff
@@ -0,0 +1,599 @@
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample code={barColumn} name="Bar vertical" layout="column" />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Stacked bars"
+          description={`Stacked bar charts break bars into smaller categories so that their relationship to the whole can be seen.
+
+Requires props: \`type="bar"\` (default) and \`stacked={true}\`
+
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample code={stackedBar} name="Stacked bars" layout="column" />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Line"
+          description={`A line graph plots numeric values for categorical data as a line that shows a progression through time.
+
+Requires props:  \`type="line"\` and \`elements\` should be set to \`type="line"\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={line} name="Line" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Combo"
+          description={`This combines a bar graph with a line graph. It is useful to see both categories and a trend or range over time.
+
+Requires props:  \`type="combo"\` and \`elements\` should be set to \`type="line"\` \`type="bar"\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={combo} name="Combo" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Biaxial"
+          description={`Biaxial graphs show two y-axis. They're used when different amounts need to be displayed per each category represented in the graph.
+
+Requires props:  \`layout="verticalBiaxial"\` (default) or  \`layout="horizontalBiaxial"\`
+`}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={biaxial} name="Biaxial" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Title & description"
+          description={`A title for the graph in case it’s not displayed elsewhere on the screen. An optional description is available if more context is needed.
+
+Requires props:  \`title\` and \`description"\`.
+`}
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={title} name="Title & description" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Tooltip"
+          columns={2}
+          description={`For showing more precise details on hover.
+
+Requires props:  \`renderTooltip\`
```

Comment:
nit: I don't think we need the "requires" word, thinking just 'Props':  will do, and then a list.